sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1179,6 +1179,59 @@ def max_deflection(self):
         else:
             return None
 
+
+    def bending_stress(self, y, m):
+        b_stress = (m*y)/self.second_moment
+        return b_stress
+
+
+    def max_bending_stress(self, y_max, m=None):
+        """Returns the maximum bending stress and its coordinates
+        in the Beam object.
+
+        Parameters
+        ==========
+        y_max : Sympifiable
+                A Symbol or a value for the maximum distance from the neutral axis
+        m : Sympifiable or Optional
+            A symbol or a value for maximum bending moment. If not given will be determined
+            by the function max_bmoment()
+        """
+        if m is None:
+            x, m = self.max_bmoment()
+        else:
+            x = self.max_bmoment()[0]
+        max_b_stress = self.bending_stress(y_max, m)
+        return ((x, y_max), max_b_stress)
+
+
+    def shear_stress(self, q, b, v):
+        shear = (q*v)/(self.second_moment*b)
```

Comment:
This equation is not quite correct. Transverse shear is a bit more complicated.

See my notes here: http://moorepants.github.io/eme150a/pages/notes-06.pdf

Reply:
I think the difficult part in it is to determine `Q` which is the first moment of area between the location where the shear stress is being calculated and the location where the shear stress is zero about the neutral (centroidal) axis.

So this value is directly taken from the user.

We can make a function to calculate it, but it would require the description(or shape) of the cross-section of the beam as there is is no general formula for it.

Reply:
We currently collect I from the user, so it would seem that we'd need to collect Q from the user. In the long run it would be ideal if we passed in a 2D shape to Beam instead of I, then all these tings can be computed automatically. This is fine for now, but it needs to be a fully documented and tested method.

Reply:
Sure, I will be completing the remaining documentation.